XmlAbstraction/XmlObject.cs

```diff
@@ -656,6 +650,8 @@ public void Delete(string elementname, string attributename)
                 {
                     throw new ArgumentException("elementname or attributename does not exist in the xml or in pending edits.");
                 }
+
+                this.HasChanged = true;
```

Comment:
Should we only set this to true only if it is not then set it to true?

Reply:
Its not really a big deal considering this is just a bool and the actual setting of it doesn't affect anything in the code. But if you want to you could do it in the property setter, but consequently will have to declare the underlying variable and not use auto property.

Reply:
Wow I had to manually rebuild the PR on the CI project page in order to build the latest commit.

Reply:
Now that I think about it we should increment the version to 1.1.2.